tests/dec/test_dec.py

```diff
@@ -191,6 +193,25 @@ def test_missing_global_photos_flag():
     assert not p.global_photos_flag()
 
 
+def test_model_aliases():
```

Comment:
Totally cosmetics but can you move this test to next to the related test `test_aliases_parsing()` and while at it follow the same name "convention?